.travis.yml

```diff
@@ -7,32 +7,41 @@ matrix:
     - os: windows
       env: BUILD_CONFIGURATION=Release
     - os: linux
-      dist: trusty
-      group: travis_lts
-      sudo: required
-      env: BUILD_CONFIGURATION=Debug
+      compiler: gcc
+      addons: &gcc9
+        apt:
+          sources: ['ubuntu-toolchain-r-test']
+          packages: ['g++-9', 'gcc-9']
+      env: GCCVER='9' CXX_COMPILER=g++-${GCCVER} CXX=${CXX_COMPILER} C_COMPILER=gcc-${GCCVER} CC=${C_COMPILER} AR=gcc-ar-${GCCVER} RANLIB=gcc-ranlib-${GCCVER} NM=gcc-nm-${GCCVER} BUILD_CONFIGURATION=Release
     - os: linux
-      dist: trusty
-      group: travis_lts
-      sudo: required
-      env: BUILD_CONFIGURATION=Release
+      compiler: gcc
+      addons: &gcc9
+        apt:
+          sources: ['ubuntu-toolchain-r-test']
+          packages: ['g++-9', 'gcc-9', 'doxygen', 'doxygen-doc', 'doxygen-latex', 'doxygen-gui', 'graphviz']
+      env: GCCVER='9' CXX_COMPILER=g++-${GCCVER} CXX=${CXX_COMPILER} C_COMPILER=gcc-${GCCVER} CC=${C_COMPILER} AR=gcc-ar-${GCCVER} RANLIB=gcc-ranlib-${GCCVER} NM=gcc-nm-${GCCVER} BUILD_CONFIGURATION=Debug
     - os: osx
       osx_image: xcode9.1
       env: BUILD_CONFIGURATION=Debug
     - os: osx
       osx_image: xcode9.1
       env: BUILD_CONFIGURATION=Release
 
-before_install:
-  - . .travis/before-install.sh
-
-install:
-  - . .travis/install.sh
```

Comment:
If we don't need those scripts anymore, it should be removed.

Reply:
yes